src/app/(dashboard)/layout.tsx

```diff
@@ -10,7 +10,7 @@ export default async function DashboardLayout({
   return (
     <html lang="en">
       <Header />
-      <div className="flex h-screen overflow-hidden">
```

Comment:
Making this comment to get credits hahaha

Reply:
lmao